local/install_deps_linux.bash

```diff
@@ -25,7 +25,9 @@ while [ "$1" != "" ]; do
 done
 
 if [ -z "$PYTHON" ]; then
-  if which python3.10 > /dev/null; then
+  if which python3.11 > /dev/null; then
+    export PYTHON='python3.11'
+  elif which python3.10 > /dev/null; then
     export PYTHON='python3.10'
   elif which python3.9 > /dev/null; then
```

Comment:
Need to install pipenv through apt

Reply:
Any reason in particular for this? We've had issues with different users having different `pipenv` versions in the past, since there are incompatibilities between versions.

Reply:
The local deps script breaks for PYTHON=python3.11 ./local/instal_deps.bash. Installing it through apt was the only solution I found (Jonathan's approach on last year's attempt)

Reply:
What's the specific breakage? CC @jonathanmetzman as well. 

I'm worried this will introduce more inconsistencies down the line.

Reply:
This is the badness that happens attempting to install pipenv through pip

![image](https://github.com/user-attachments/assets/9f08fd16-9512-42fc-8126-f547ad2a9d6a)